prediction_market_agent_tooling/markets/polymarket/copy_trading.py

```
@@ -0,0 +1,451 @@
+    replicated_amount: float
+    skipped: bool = False
+    skip_reason: str | None = None
+
+
+class PolymarketCopyTrader:
+    def __init__(
+        self,
+        target_address: ChecksumAddress,
+        api_keys: APIKeys,
+        copy_ratio: float = 1.0,
+        min_trade_size: USD = USD(1.0),
```

Ruff (0.15.9)

[warning] Do not perform function call `USD` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)